src/Site/Controller/Upload.php

```diff
@@ -53,33 +53,28 @@ public function receive(Application $app, $appType, $mediaType)  { // e.g. 'lf',
                     $response = $api->sfChecks_uploadFile($mediaType, $tmpFilePath);
                 } elseif ($appType == 'lf-lexicon') {
                     $api = new Sf($app);
+                    $jsonResult = array(
+                        'orderedParams' => array(
+                            $mediaType,
+                            $tmpFilePath
+                        ),
+                        'projectId' => $_POST['projectId']
```

Comment:
As of this morning there was only one failing e2e test. Now there are five, since the fix involved requiring a `projectId`. They're all in `lexicon-new-project.spec.js`. I think they're probably all related to this line, where we try to access projectId, which may not exist (but ought to).